crt_portal/static/js/form_letter.js

```diff
@@ -0,0 +1,59 @@
+(function(root) {
+  var modal = document.getElementById('intake_template');
+
+  var contact = document.getElementById('contact_complainant');
+  contact.onclick = function(event) {
+    event.preventDefault();
+    if (modal.getAttribute('hidden') !== null) {
+      root.CRT.openModal(modal);
+    } else {
+      root.CRT.closeModal(modal);
+    }
+  };
+
+  var cancel_modal = document.getElementById('intake_template_cancel');
+  root.CRT.cancelModal(modal, cancel_modal);
+
+  var copy = document.getElementById('intake_copy');
+  var print = document.getElementById('intake_print');
+  var letter = document.getElementById('intake_letter');
+  var description = document.getElementById('intake_description');
+  var options = document.getElementById('intake_select');
+  options.onchange = function(event) {
+    event.preventDefault();
+    var index = event.target.selectedIndex;
+    var option = event.target.options[index];
+    description.innerHTML = option.dataset['description'] || '(select a response template)';
+    letter.innerHTML = option.dataset['content'] || '';
+    if (index >= 1) {
+      copy.removeAttribute('disabled');
+      print.removeAttribute('disabled');
+    } else {
+      copy.setAttribute('disabled', 'disabled');
+      print.setAttribute('disabled', 'disabled');
+    }
+  };
+
+  copy.onclick = function(event) {
```

Comment:
I wonder if using `addEventListener` here instead would be better, because then we could also attach this set of actions to the `keydown` listener for users using the keyboard to trigger this?

Reply:
Ooh yes. Thank you, great catch.

Reply:
I ended up keeping `copy.onclick` because in testing, `keydown` would capture tabbing and it was confusing. That being said, open/cancel now respond to `keydown` as well. I'll run this past Jacklynn. Thanks!